slsim/lens.py

```diff
@@ -558,7 +577,9 @@ def source_light_model_lenstronomy(self, band=None):
         """
         source_models = {}
         all_source_kwarg_dict = {}
-        center_source = self.source_position
+        center_source = self.source.extended_source_position(
```

Comment:
does this work/not crash if we only have a point source without an extended image?

Reply:
I moved this inside the if statement.